src/audio/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ if(NOT BUILD_HOST)
 		pipeline_static.c
 		component.c
 		buffer.c
+		selector.c
```

Comment:
channel_selector.c ?

Reply:
@lbetlej The commit message should contain a verb. So, basically your patch adds support for a channel selector? 

Then the commit message should be:

component: Add support for channel selector